Nodejs/Product/Npm/SPI/NpmCommander.cs

```diff
@@ -178,24 +174,16 @@ private class NmpCommandRunner : IDisposable {
             return type;
         }
 
-        private async Task<bool> UninstallPackageAsync(string packageName, bool global) {
-            return await DoCommandExecute(true,
+        public Task<bool> UninstallPackageAsync(string packageName) {
```

Comment:
Git doesn't make it clear, but this is the result of merging the three `UninstallPackageAsync` functions.

First I deleted `public UninstallGlobalPackageAsync`. This left `private UninstallPackageAsync` function with a single caller that only forwards its arguments. So then I merged the `public UninstallPackageAsync` with the `private UninstallPackageAsync`.